### DISCUSSION TITLE
blog/cloud/2024-07-11-docker-spring-boot-deploy/

### DISCUSSION THREAD

**Discussion by giscus** [mingdodev/mingdodev.github.io#19] [Announcements]
# blog/cloud/2024-07-11-docker-spring-boot-deploy/

moonshot

https://mingdodev.github.io/blog/cloud/2024-07-11-docker-spring-boot-deploy/

**Comment by ajroot5685** [comment]
사이드 프로젝트 수준에서는 DB를 분리시키지 않고 Docker 컨테이너에 띄우는 것만으로도 충분하지만, 현업에서는 확실히 도커로는 띄우면 안될 것 같네요.. 이게 DevOps가 필요한 이유겠죠 😢

**Reply by mingdodev** [reply]
오! 이 고민으로 DevOps가 필요한 이유에 대해서도 생각해 볼 수 있었네요~~
